stable/sumologic-fluentd/templates/daemonset.yaml

```diff
@@ -13,6 +13,10 @@ spec:
       name: {{ template "fullname" . }}
       labels:
         app: {{ template "fullname" . }}
+      {{- if .Values.daemonset.podAnnotations }}
```

Comment:
Nit: the `if` and `else` statements should be indented only one level less than the block they wrap.